projects/organization-management/src/app/store/users/users.effects.ts

```diff
@@ -1,24 +1,68 @@
 import { Injectable } from '@angular/core';
 import { Actions, Effect, ofType } from '@ngrx/effects';
-import { exhaustMap, map } from 'rxjs/operators';
+import { Store, select } from '@ngrx/store';
+import { TranslateService } from '@ngx-translate/core';
+import { debounceTime, exhaustMap, map, mapTo, withLatestFrom } from 'rxjs/operators';
 
-import { mapErrorToAction } from 'ish-core/utils/operators';
+import { selectRouteParam } from 'ish-core/store/router';
+import { UserActionTypes } from 'ish-core/store/user';
+import { SetBreadcrumbData } from 'ish-core/store/viewconf';
+import { mapErrorToAction, whenTruthy } from 'ish-core/utils/operators';
 
 import { UsersService } from '../../services/users/users.service';
 
 import * as actions from './users.actions';
+import { getCurrentUser } from './users.selectors';
 
 @Injectable()
 export class UsersEffects {
-  constructor(private actions$: Actions, private usersService: UsersService) {}
+  constructor(
+    private actions$: Actions,
+    private usersService: UsersService,
+    private store: Store,
+    private translateService: TranslateService
+  ) {}
 
   @Effect()
   loadUsers$ = this.actions$.pipe(
     ofType<actions.LoadUsers>(actions.UsersActionTypes.LoadUsers),
     exhaustMap(() =>
-      this.usersService
-        .getUsers()
-        .pipe(map(users => new actions.LoadUsersSuccess({ users }), mapErrorToAction(actions.LoadUsersFail)))
+      this.usersService.getUsers().pipe(
+        map(users => new actions.LoadUsersSuccess({ users })),
+        mapErrorToAction(actions.LoadUsersFail)
+      )
     )
   );
+
+  @Effect()
+  loadDetailedUser$ = this.store.pipe(
+    select(selectRouteParam('B2BCustomerLogin')),
+    whenTruthy(),
+    debounceTime(0), // todo: refactor this race condition?
+    exhaustMap(login =>
+      this.usersService.getUser(login).pipe(
+        map(user => new actions.LoadUsersSuccess({ users: [user] })),
+        mapErrorToAction(actions.LoadUsersFail)
+      )
+    )
+  );
+
+  @Effect()
+  setUserDetailBreadcrumb$ = this.store.pipe(
```

Comment:
this effect doesn`t work properly if the user switches the language, e.g. if you change the language from english to german the english breadcrumb persists.
This is a general issue and also occurs for quotes.
btw, wishlists doesn`t have this `details` prefix, why do we make things so complicated?


this problem occurs also for quotes
wishlists